### PR TITLE
Add Piantor Pro

### PR DESCRIPTION
**Pull request by l4u**
## Description

Add Piantor Pro

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/beekeeb/piantor_pro/info.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0xBEEB"
+    },
+    "split": {
+        "enabled": true
+    },
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
```
Comment (dunk2k) [inline keyboards/beekeeb/piantor_pro/info.json:31]: `0, 0` is default bootmagic co-ordinate 
```suggestion
```

File: keyboards/beekeeb/piantor_pro/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Piantor Pro
+
+![Piantor Pro](https://i.imgur.com/UPRI64c.jpg)
```
Comment (dunk2k) [inline keyboards/beekeeb/piantor_pro/readme.md:3]:
```suggestion
![Piantor Pro](https://i.imgur.com/UPRI64ch.jpg)
```

File: keyboards/beekeeb/piantor_pro/readme.md
Comment (dunk2k) [inline keyboards/beekeeb/piantor_pro/readme.md:1]: please add flash example

File: keyboards/beekeeb/piantor_pro/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+A 42 or 36 key hotswappable keyboard with RP2040.
```
Comment (dunk2k) [inline keyboards/beekeeb/piantor_pro/readme.md:5]: 36 key configuration not defined, could it be added or `36 key` removed

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/beekeeb/piantor_pro/config.h [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
```
Comment (waffle87) [inline keyboards/beekeeb/piantor_pro/config.h:21]:
```suggestion
```

**Comment by dunk2k** [comment]
@l4u 
very quick question: as `USB_VBUS_PIN` is defined in `config.h`, is there a particular RP2040 based controller/development board this keyboard is built with?

**Comment by l4u** [comment]
@dunk2k For this variant, it's not built with a dev board.

**Review by dunk2k — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**